launcher/ui/widgets/ModFilterWidget.cpp

```diff
@@ -279,16 +279,17 @@ void ModFilterWidget::onSideFilterChanged()
 {
     QString side;
 
-    if (ui->clientSide->isChecked() != ui->serverSide->isChecked()) {
-        if (ui->clientSide->isChecked())
-            side = "client";
-        else
-            side = "server";
+    if (ui->clientSide->isChecked() && !ui->serverSide->isChecked()) {
+        side = "client";
+    } else if (!ui->clientSide->isChecked() && ui->serverSide->isChecked()) {
+        side = "server";
+    } else if (ui->clientSide->isChecked() && ui->serverSide->isChecked()) {
+        side = "both";
     } else {
-        // both are checked or none are checked; in either case no filtering will happen
-        side = "";
+        side = "none";
```

Comment:
use an empty string, please as this is used by Curseforge.
```suggestion
        side = "";
```

Reply:
whoops sorry, i missed that!